### DISCUSSION TITLE
Aqara FP1 and MA

### DISCUSSION THREAD

**Discussion by johnjoemorgan** [jseidl/magic-areas#281] [Q&A] [answered]
Hi

Does MA handle the Aqara FP1 as a 'normal' binary sensor?  That is "home " & "arriving" are seen as "on" and "away" and "leaving" are seen as "off"?

Cheers

John

**Comment by pservais** [comment, marked as answer]
You can use the 'presence' binary sensor of the FP1, thats what I'm using.

**Comment by jseidl** [comment]
You can also use any sensor and add the required state you're looking for (home, arriving)
